webapp/src/services/events/events.service.js

```diff
@@ -12,6 +12,6 @@ export function getEvents() {
         })
         .catch(function (error) {
             // handle error
-            return { error: error };
+            return [], { error: error };
```

Comment:
Return [] ?

Reply:
It was trying to render a with a null, i have since fixed the backend so that that won't happen. I will move the check to the controller